analytics/dev/.env.docker-compose

```diff
@@ -10,3 +10,4 @@ CELERY_BROKER_URL=redis://localhost:6379/0
 SECRET_KEY=deadbeef
 GITLAB_ENDPOINT="http://fakeurl"
 GITLAB_TOKEN="bar"
+PROMETHEUS_URL="http://localhost:9090
```

Comment:
Does this work with the quoting? And I think it needs to be added to the native version of this file too.

Reply:
Whoops, that's a typo. Also, there's no (existing) native version of this file.

Reply:
Oh right, that's my PR 🤦‍♂️